src/EventCalendar.js

```diff
@@ -65,120 +104,526 @@ class EventCalendar {
+                    break;
+                case 'clear':
+                default:
+                    _self.styleClear(this);
+                    break;
+            }
+        });
     }
 
+    /**
+     * Triggered when navigating through or making any changes to the calendar. Takes the dates stored in the 
+     * `datesToAdd` and `datesToDel` arrays as well as the currently stored recurrence pattern and paints the dates
+     * appropriately. Also triggers the updating of the review panel when finished.
+     * 
+     * @param {moment} month A moment object representing the currently viewed month
+     * @param {string} method `repeat-on` will paint the dates, `clear` will unpaint them
+     */
+    paintMonth (month, method) {
+        let _self = this;
+
+        // Unpaint the entire month, used to catch any dates resetting to neutral
+        _self.styleClearAll();
+
+        // Recurrence pattern to paint for the displayed month
+        if (_self.recurPattern != null) {
+            _self.paintRepeatPattern(method);
+        }
+
+        // Exceptions to paint as to-add
+        if (_self.clndr.options.extras.datesToAdd.length) {
+            const datesToAdd = _self.clndr.options.extras.datesToAdd
+                                .filter(EventCalendar.isInYear.bind(this, month))
+                                .filter(EventCalendar.isInMonth.bind(this, month));
+
+            $.each(datesToAdd, function() {
+                _self.styleToAdd(this);
+            });
+        }
+
+        // Exceptions to paint as to-delete
+        if (_self.clndr.options.extras.datesToDel.length) {
+            const datesToDel = _self.clndr.options.extras.datesToDel
+                                .filter(EventCalendar.isInYear.bind(this, month))
+                                .filter(EventCalendar.isInMonth.bind(this, month));
+
+            $.each(datesToDel, function() {
+                _self.styleToDel(this);
+            });
+        }
+
+        _self.updateReview();
+    }
+
+    /**
+     * Style a date which should be added to the list of active dates
+     * 
+     * @param {moment} target The date to style
+     */
+    styleToAdd (target) {
+        let _self = this,
+            $elem = _self.clndr.element.find('[data-day="' + target.format(_self.dateFormat) + '"]'),
+            ariaLabel = 'Selected. Event will repeat on ' + target.format(_self.dateFormatLong);
+
+        $elem.attr('aria-label', ariaLabel)
+             .parent()
+             .addClass('event-add');
+
+        // Announce the new status through a live region (instead of the previous status)
+        _self.updateLiveRegion(ariaLabel);
+    }
+
+    /**
+     * Make screen readers announce the `message` by updating a hidden aria-live region within the template
+     * 
+     * @param {string} message The text to be announced by the screen reader
+     */
+    updateLiveRegion (message) {
+        let _self = this;
+
+        _self.$ariaLiveRegion.text(message);
```

Comment:
jQuery affords us some laziness here by allowing us to pass it `undefined` to (I've not checked) a lot of it's chain-able methods. So we could leave this as it is, however It's worth noting that `_self.$ariaLiveRegion` _could_ be `undefined` here, so if down the line we were to add anything that expected the return value of `_self.$ariaLiveRegion.text(message)` to be a reference to a DOM element we might trip over.

Something like this would break for example `_self.$ariaLiveRegion.text(message)[0].focus()` if `_self.$ariaLiveRegion` was `undefined`

We are probably safe to leave this as it is, but I thought it's worth mentioning.

If we wanted to be jQuery agnostic we'd probably want to add a quick early return inside these methods along the lines of, so if this method grows we can safely assume past a certain point that the property will contain what we expect.

```
if (_self.$ariaLiveRegion === undefined) {
    return;
}
```